jobs/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
```

Comment:
Would you mind putting
`layout: static`

and removing the author. 
Since this part is pretty standard and shouldn't look like a blog post.